Customizing/global/lang/ilias_de.lang.local

```diff
@@ -1540,3 +1540,4 @@ gev#:#gev_entry_date_invalid#:#Das angegebene Datum ist ungültig oder falsch fo
 gev#:#gev_my_assessments#:#Assessmenttermine
 gev#:#gev_all_assessments#:#Alle Assessmenttermine
 gev#:#no_wbt_found_for_ref#:#Kein Training zur Frage Nr. %s gefunden.
+gev#:#placeholder#:#Test
```

Comment:
Placeholder text for the select box. I remember it on the way and would change it to something usefull.